src/tiled/commanddialog.cpp

```diff
@@ -45,6 +45,8 @@ CommandDialog::CommandDialog(QWidget *parent)
 
     setWindowTitle(tr("Edit Commands"));
     Utils::restoreGeometry(this);
+
+    setWindowFlags(Qt::Window | Qt::WindowTitleHint | Qt::CustomizeWindowHint);
```

Comment:
Please instead of removing the close button, which I'm sure will confuse some people, change the `accept` method to a `closeEvent` override. It didn't actually make sense to keep it as `accept` when you can only just close the dialog anyway. You'll also need to connect the "Close" button at the bottom to the "close()" slot instead of the "accept()" slot.